docs/content/getting-started/tutorial/webapp-add-database/snippets/db-container.bicep

```
@@ -0,0 +1,62 @@
+import radius as radius
+
+param environmentId string
```

```suggestion
param environment string
```

For all the bicep files in this tutorial, we need to make this change - after a bit of discussion, the final answer for this param was `environment`. 
We also need to make this change down on the app resources and connector resources as well.

confirmed by @jkotalik when I rudely interrupted him just now.